usmqe_tests/conftest.py

```diff
@@ -40,3 +42,49 @@ def logger_testcase(request):
     LOGGER.testStart(get_name(str(request.node)))
     yield
     LOGGER.testEnd()
+
+
+@pytest.fixture(
+    params=[{
+        "name": "Tom Hardy",
+        "username": "thardy",
+        "email": "thardy@tendrl.org",
+        "role": "admin",
+        "password": "pass1234",
+        "password_confirmation": "pass1234"}])
```

Comment:
Ouch :) Could we add comment with link to https://github.com/Tendrl/api/issues/106 - strictly speaking, this is a workaround.